core/parameters/res/runtime_configs/145.yaml

```diff
@@ -0,0 +1 @@
+storage_contract_code_burnt_amount_per_byte: { old: 0, new: 100_000_000_000_000_000_000}
```

Comment:
Since the cost is quite high, does it implicitly impose a smaller limit on the contract size?